cmd/neofs-cli/modules/control.go

```diff
@@ -37,6 +41,16 @@ const (
 	netmapStatusOffline = "offline"
 )
 
+// control healthcheck flags
+const (
+	healthcheckIRFlag = "ir"
+)
+
+// control healthcheck vars
+var (
+	healthCheckIRVar bool
```

Comment:
I would rather remove `( )` if block consists one variable or one constant.

Reply:
This is how I usually do, but I indicated in the comment the semantic grouping of names.